generatedocs/data/actions/thinktime/examples.md

```diff
@@ -16,6 +18,8 @@
 
 #### ThinkTime constant
 
+This would simulate a think time of 5 seconds.
```

Comment:
Maybe change to "This simulates a think time of 10 to 15 seconds."?

Reply:
changed